todo_app.js

```diff
@@ -1,5 +1,108 @@
 
-    var todosListEl;
+
+
+function TodoApp() {
+    //your code to run since DOM is loaded and ready;
+    this.initApp();
+}
+
+document.addEventListener("DOMContentLoaded", function() {
+    var todoApp = new TodoApp();
+});
+
+
+TodoApp.prototype.initApp = function() {
+    this._todosListEl = document.getElementById('todos'); // ������ �� ��� ������� ������ �������
+    var addButton = document.querySelector('div > input:last-child');
+    addButton.addEventListener('click', this.addTodo.bind(this));
+    TodoManager.init();
+    this.renderList();
+};
+
+
+TodoApp.prototype.checker = function (event) {
+    var index = event.target.getAttribute('data-index');
+    event.target.parentNode.parentNode.classList.toggle('completed');
+    TodoManager.toggleTodo(index);
+};
+
+
+TodoApp.prototype.removeTodo = function (event) {
+    var index = event.target.getAttribute('data-index');
+    TodoManager.removeTodo(index);
+    this.renderList();
+};
+
+
+TodoApp.prototype.renderList = function() {
+    // renderTodos(todosFromLocalStorage);
+    var todoList = TodoManager.loadTodos();
+    this._renderedTodoList = todoList.map(function(todo, index) {
+        return this.renderTodo(todo, index);
+    }, this);
+
+    this._todosListEl.innerHTML = this._renderedTodoList.join('');
+
+    this.handlersToCheckboxes();
```

Comment:
You can merge both functions to one and name it bindListControls. Because they both bind handlers to elements inside list container and at the same moment.